components/automate-chef-io/content/docs/eas_applications.md

```diff
@@ -0,0 +1,161 @@
+
+## Health Checks
+To maximize the utility of the Chef Automate EAS Applications feature, it is recommended to implement meaningful health check hooks for your services. When a health check hook returns a status other than “OK”, Chef Automate will display the output of the hook along with the status, so it is recommended to include useful debugging information in the output of the health checks when returning a “critical”, “warning”, or “unknown” status result. Further information about writing habitat health check hooks can be found in the habitat documentation.
+
+## Habitat Named Service Groups
+To minimize Habitat network gossip overhead, it is currently recommended that each deployment (as defined above) use a separate Habitat network. When using this architecture, the Habitat named service groups feature is only useful in some edge case situations, such as when running multiple discrete clusters of the same software within a single application. Therefore, in most cases, it is recommended to not use this feature and leave the service group name at the default setting (default).
+
+## Service Groups in Chef Automate
+The Chef Automate interface will group any services with the same package name, environment, service group name, and application together in the main table view. This allows you to quickly understand the status of your habitat services even when scaling out to large numbers of services, with detailed reporting of each individual service shown in a card on the right-hand sidebar. In the table view, the status reported for a service group will reflect the “most critical” status reported by any individual instance of that service (e.g. if one instance is in “critical” status, the entire group is in “critical” status). By default, the table rows are sorted by percentage of OK services from the smallest to the largest. The sidebar cards are sorted by criticality, from the “most critical” status to the “least critical” status (“critical,” “warning,” “unknown”,  and “OK”). The most critical issues are shown at the top of the list, allowing you to quickly determine the next steps for remediation.
+
+## Requirements and Capacity Planning
+Based on the latest testing, the EAS Applications system can scale up to 15,000 services on a system with 4(v)CPUs, 16GB of RAM, and SSD disks.
+
+You must use habitat 0.83 or greater.
+
+At a larger scale, the subsystems responsible for processing applications data will consume significant CPU resources. If the Compliance and Infrastructure visibility features of Automate will also be used on this Chef Automate instance, the host system will need additional compute and memory capacity.
+
+## Configuring Chef Habitat to Send Events to Chef Automate
+In the Chef Automate UI, create an API token as described in the Chef Automate documentation. Save this token somewhere safe.
+Please see (https://www.habitat.sh/docs/habitat-cli/#hab-sup-run "habitat documentation") for more habitat options.
```

Comment:
should this "please see.." sentence be down further, like under the section that says to launch with options?